lib/plugins/package/package.js

```diff
@@ -55,6 +55,22 @@ class Package {
     };
 
     this.hooks = {
+      'initialize': () => {
+        const useIncludeExclude = (packageConfig = {}) =>
+          packageConfig.include || packageConfig.exclude;
+        const servicePackage = this.serverless.service.package || {};
+        if (
+          useIncludeExclude(servicePackage) ||
+          Object.values(this.serverless.service.functions).some(func =>
+            useIncludeExclude(func.package)
+          )
+        ) {
+          this.serverless._logDeprecation(
+            'NEW_PACKAGE_PATTERNS',
+            'Starting with next major version, the "include" and "exclude" options will be removed.'
```

Comment:
Let's reword it to: _Support for`package.include` and `package.exclude` will be removed with next major release. Please use `package.patterns` instead_